docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -271,25 +261,57 @@ spec:
 ```
 
 
-## RBAC for Sessions
+## Teleport Resources
+
+RBACs lets teams limit what resources are available to teleport users. This can be
+helpful if you don't want regular users editing SSO `auth_connector` or creating and
+editing new roles `role`.
```

Comment:
```suggestion
RBAC lets teams limit what resources are available to Teleport users. This can be
helpful if you don't want regular users editing SSO (`auth_connector`), for example,
or creating and editing new roles (`role`).
```

Reply:
( Added this myself )